pom.xml

```diff
@@ -89,6 +87,12 @@
 				<version>${hapifhirVersion}</version>
 			</dependency>
 
+            <dependency>
```

Comment:
nit: Please fix indent (replaces spaces with tabs as the rest of the file).

Reply:
xml formatting would be helpful - for some reason IntelliJ was being really strange about the whitespace in this file.